### PR TITLE
Adding link to post from modal card

### PR DESCRIPTION
**Pull request by lakshyaoberoi**
GitHub Issue (If applicable): #90 

## PR Type

What kind of change does this PR introduce?
- Feature

## What is the current behavior?

Currently, the modal cards show information extracted from the source only, with no link to access the source itself.

## What is the new behavior?

Added a link button that is displayed on each modal except twitter (no API in the modal - described in issue) which directs the user to the source of the information.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current upstream repo
- [ ] Added new tests to the test suite for the feature
- [ ] Tested the full test suite, and ensured that the feature does not break current build.
- [x] Docs have been added/updated which fit (for bug fixes / features)
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal)
- [x] Checked licensing of Frameworks/Libraries (if applicable) 

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by lakshyaoberoi** [comment]
> @MarioSinovcic ? does this conflict with you?

Peer programming with @MarioSinovcic

**Comment by baja156** [comment]
> > @MarioSinovcic ? does this conflict with you?
> 
> Peer programming with @MarioSinovcic

Cool beans!

File: client/src/components/FeedrModal.jsx [hunk 1]
```diff
@@ -46,10 +49,51 @@ const useStyles = makeStyles(theme => ({
...
+    alignItems: 'left',
+    justifyContent: 'left',
+  },
 }));
 
 export const FeedrModal = props => {
   const classes = useStyles();
+
+  if (props.media !== 'twitter') {
+    var externalLink;
```
Comment (FocalChord) [inline client/src/components/FeedrModal.jsx:63]: Could we change everywhere we use var to let/const. For consistency sake :)

Comment (lakshyaoberoi) [reply client/src/components/FeedrModal.jsx]: done

Comment (lakshyaoberoi) [reply client/src/components/FeedrModal.jsx]: was getting errors when using let/const for the renderLink component.

**Comment by Dinith1** [comment]
There's some conflicts 😿

**Comment by baja156** [comment]
Pls resolve conflicts before merge

**Pull request merged**